console.go

```diff
@@ -22,6 +22,10 @@ func (l *writerLogger) Writer(sev Severity) io.Writer {
 	return nil
 }
 
+func (l *writerLogger) SetSeverity(sev Severity) {
+	l.sev = sev
```

Comment:
This write races with the read in `Writer`. We should either make the read and write atomic or add a comment that this is not safe to call concurrently with `Writer`.